ci/package.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd /src
+
+. ./ci/install-build-deps.sh
+
+
+mkdir -p packages
+
+
+case "$CONTAINER_OS" in
+    'centos:7')
+        case "$ARCH" in
+            'arm32v7'|'aarch64')
+                echo 'Cross-compilation on CentOS 7 is not supported' >&2
```

Comment:
Add more loggings can be useful in case of issue.

Reply:
The log already tells you what the problem is. What more do you want it to say?